npm-shrinkwrap.json

```diff
@@ -8746,14 +8739,39 @@
       }
     },
     "resin-corvus": {
-      "version": "1.0.0-beta.26",
-      "from": "resin-corvus@1.0.0-beta.26",
-      "resolved": "https://registry.npmjs.org/resin-corvus/-/resin-corvus-1.0.0-beta.26.tgz",
+      "version": "1.0.0-beta.27",
+      "from": "resin-io-modules/resin-corvus#v1.0.0-beta.28",
```

Comment:
Ahh, looks like this is because the version in the [git tag](https://github.com/resin-io-modules/resin-corvus/tree/v1.0.0-beta.28) doesn't match the version in [package.json](https://github.com/resin-io-modules/resin-corvus/blob/v1.0.0-beta.28/package.json)